runtime/ompstub/ompstubs.c

```diff
@@ -590,6 +590,12 @@ omp_get_wtick_(void)
   return 365. * 86400.;
 }
 
+int
+omp_in_final(void)
+{
+  return 1;
```

Comment:
Is the value returned here important?

Reply:
No, it is not important. Shall I leave the function body empty?

Reply:
No need. I was thinking whether 0 is a better value. But if value is not important then not required.